### PR TITLE
fix: prevent esc key close login dialog

### PR DESCRIPTION
**Pull request by ggstone0523**
Resolve: lablup/backend.ai-webui #1168 

- This bug caused by auto closing backend-ai-dialog when press esc button
- add 'escapeKeyAction' attribute to mwc-dialog in backend-ai-dialog
- change 'escapeKeyAction' attribute value from 'close'  to '' only when open cluster dialog

**Currently, If you press the esc key right after the 'login page' opens, you can see the 'cluster dialog'.**
PR5_review0_1

### CODE REVIEW

**Comment by ggstone0523** [comment]
### The revision was completed.

1.  show login dialog when press X button

**Comment by Sujin-Kim1** [comment]
I think `escapeAction` is not necessary because it works well without that. 
It's trivial, but from the second time, if you press esc to make the login dialog disappear, the blank page will appear again.

**Comment by ggstone0523** [comment]
In my development environment(safari, chrome),  cluster dialog disappear when press esc key if remove `escapeKeyAction`.
and, i avoid closing the login dialog with the esc key when the login dialog appears second time or press esc key late (because if press esc key late, cluster dialog did not appear)

**Comment by ggstone0523** [comment]
### The revision was completed.

1. prevent login dialog closed by esc key pressing
2. add 'preventEscKeyClose' attribute to backend-ai-dialog

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM.

File: src/components/backend-ai-dialog.ts [hunk 1]
```diff
@@ -160,14 +162,28 @@ export default class BackendAiDialog extends LitElement {
    * Open a dialog.
    */
   show() {
+    this.checkPreventEscapeKeyClose();
     this.dialog.show();
   }
```
Comment (lizable) [inline src/components/backend-ai-dialog.ts]: Why don't we just use string value (`close`) as default value which is the same way the `mwc-dialog` supports?

**Review by lizable — CHANGES_REQUESTED** [review]
I'm not sure about whether we should need boolean property instead of using the internal value that mwc-dialog supports.
IMHO, using escapeKeyAction is better since we don't need to assign an extra function for value checking.
What do you think of this approach @ggstone0523 @Sujin-Kim1 ?

**Comment by ggstone0523** [comment]
I think is good!

**Comment by ggstone0523** [comment]
### The revision was completed by reflecting the comments.

- remove preventEscapeKeyClose attribute and append escapeKeyAction attribute in backend-ai-dialog

**Comment by ggstone0523** [comment]
The revision was completed

- remove cluster dialog protection from disappear

**Comment by ggstone0523** [comment]
### The revision was completed.

- add 'escapeKeyAction' attribute to block-panel

**Review by lizable — APPROVED** [review]
LGTM.

**Pull request merged**